server/app/modules/MainModule.java

```diff
@@ -22,10 +26,15 @@ public class MainModule extends AbstractModule {
 
   @Override
   public void configure() {
-    // Use the system clock as the default implementation of Clock
-    bind(Clock.class).toInstance(Clock.system(ZoneId.of("America/Los_Angeles")));
     bind(ProgramService.class).to(ProgramServiceImpl.class);
     bind(QuestionService.class).to(QuestionServiceImpl.class);
     bind(ApplicantService.class).to(ApplicantServiceImpl.class);
   }
+
+  @Provides
+  public Clock provideClock(Config config) {
+    ZoneId zoneId = ZoneId.of(checkNotNull(config).getString("java.time.zoneid"));
```

Comment:
Do you know how this java.time.zoneid param works? I'm surprised you have to explicitly read it versus java handling it for us.

Reply:
This actually isn't something that Java handles by default, though it's name definitely implies it. I believe the original naming was due to the fact that it's setting the string value for a `java.time.ZoneId` instance :/.

Reply:
I'd suggest we remove that confusion if it's in scope for this PR.  Does anything use it currently?

Reply:
As I understand it, no. To be extra-cautious, I'd suggest we rename the config value in this PR and keep the environment-variable override as `JAVA_TIME_ZONE_ID` until we've confirmed that no deployments set that. Once that's confirmed, I'll also rename the environment variable.

Latest commit incorporates your change.

Reply:
Thanks for being cautious but in this case I think you can make the more expedient change.

Seattle isn't setting it: https://github.com/seattle-uat/civiform-deploy/blob/main/infra/containers.yaml#L123-L161
Rochester isn't setting it either since it's not configurable via terraform at the moment.

Reply:
Done.